doc/source/whatsnew/v0.18.2.txt

```diff
@@ -19,10 +19,66 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0182.enhancements.read_csv_dupe_col_names:
 
+Duplicate column names are now supported in ``pd.read_csv()`` whether they are in the file
+or passed in as the `names` parameter (:issue:`7160`, :issue:`9424`)
 
+.. ipython :: python
 
+   data = '0,1,2\n3,4,5'
+   names = ['a', 'b', 'a']
 
+Previous behaviour:
+
+.. code-block:: ipython
+
+   In [2]: pd.read_csv(StringIO(data), names=names)
+   Out[2]:
+      a  b  a
+   0  2  1  2
+   1  5  4  5
+
+The first 'a' column contains the same data as the second 'a' column, when it should have
+contained the array ``[0, 3]``.
+
+New behaviour:
+
+.. code-block:: ipython
+
```

Comment:
no this is a ``code-block``, not an ``ipython:: python``, we always want this to run code if possible. So the previous are almost always ``code-block`` and new ``ipython``